app/models/deck.rb

```diff
@@ -7,4 +7,31 @@ class Deck < ActiveRecord::Base
   validates :title, presence: true, uniqueness: {
               scope: :user_id
             }
+
+  def add_cards(cards)
+    deck_cards = cards.map do |name, quantity|
+      card = Card.find_by(name: name)
+      deck_card = DeckCard.new(card_id: card.id, quantity: quantity)
+    end
+    self.deck_cards = self.deck_cards + deck_cards
+  end
+
+  def sample_deck
```

Comment:
I'd prefer `sample_hand` as a name as it generates a sample hand of 7 cards but this is functionally correct.